power/03_influx.yaml

```diff
@@ -0,0 +1,98 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: power
+  name: influxdb
+  labels:
+    component: influxdb
+    app: influxdb
+spec:
+  clusterIP: None
+  ports:
+  - port: 8086
+    name: server
+  selector:
+    component: influxdb
```

Comment:
if you do get rid of the component label, you will need to update this selector.

Reply:
done.